src/ui/Notebook.ts

```diff
@@ -187,12 +189,13 @@ export class Notebook {
       };
 
       await invoke("run_cell", { kernelId: this.kernelId, code, onEvent });
-      this.state.setOutput(cellId, { status, output, displays });
+      this.state.setOutput(cellId, { status, output, displays, durationMs: performance.now() - startTime });
     } catch (error: any) {
       this.state.setOutput(cellId, {
         status: "error",
         output: error,
         displays: {},
+        durationMs: performance.now() - startTime,
```

Comment:
we need to decide if we're okay to calculate this client-side or if we need to get it from the server/kernel. imo client-side is a good start because it's close to what the user understands the execution time to be (incl latency)